x-pack/plugins/security/server/authentication/authentication_service.ts

```diff
@@ -93,13 +93,7 @@ export class AuthenticationService {
         });
       }
 
-      let authenticationResult;
-      try {
-        authenticationResult = await this.authenticator.authenticate(request);
-      } catch (err) {
-        this.logger.error(err);
-        return response.internalError();
-      }
+      const authenticationResult = await this.authenticator.authenticate(request);
```

Comment:
__question:__ I'm pretty sure the answer is `yes`, but just to be sure: with this change we'll still never ever leak error details to the client, only vague `Internal Server Error`-like message, right?

Reply:
Exactly! The default behaviour in the HTTP service catches the error, logs it, and replies `500 - Internal Server Error` in the message (no actual error message leaked).

https://github.com/elastic/kibana/blob/af75079a31e98c8a6c0141393617b39eeb39d9bf/src/core/server/http/router/router.ts#L271-L287